newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageHeader.js

```diff
@@ -39,16 +44,32 @@ export const HomePageHeader = ({
           noMargin
           expand
         >
-          <IconButton
-            size="small"
-            id="main-toolbar-project-manager-button"
-            onClick={onOpenProjectManager}
-            tooltip={t`Project Manager`}
-            color="default"
-            disabled={!hasProject}
-          >
-            <ProjectManager />
-          </IconButton>
+          <Column noMargin>
+            <Line noMargin>
+              <IconButton
+                size="small"
+                id="main-toolbar-project-manager-button"
+                onClick={onOpenProjectManager}
+                tooltip={t`Project Manager`}
+                color="default"
+                disabled={!hasProject}
+              >
+                <ProjectManager />
+              </IconButton>
+              {!!hasProject && (
```

Comment:
Is the `!!` needed?

Reply:
probably not, but I think it doesn't hurt having it!